lms/templates/admin/instance.html.jinja2

```diff
@@ -136,7 +136,7 @@
 
 
     {% call macros.field_body(label="Move to organization") %}
-        <form method="POST" action="{{ request.route_url("admin.instance.id", id_=instance.id) }}">
+        <form method="POST" action="{{ request.route_url("admin.instance.move_org", id_=instance.id) }}">
```

Comment:
There are many alternatives to this fix that still allow to use the same endpoint but I reckon this is the easiest to implement and reason about.